torch/_dynamo/symbolic_convert.py

```diff
@@ -1174,6 +1181,8 @@ class InstructionTranslatorBase(
     parent: Optional[InstructionTranslatorBase]
     debug_locals: list[tuple[VariableTracker, list[VariableTracker]]]
     package: Optional[CompilePackage]
+    latest_bytecode_queue: deque[str] = deque(maxlen=20)
```

Comment:
This is improperly initialized - every instance of InstructionTranslatorBase will refer to the same deque - this is why you see the 13, 20 problem.

Reply:
Fixed! Thank you!